app/registries/foreman/plugin/graphql_plugin_fields.rb

```diff
@@ -6,7 +6,11 @@ module GraphqlPluginFields
       module ClassMethods
         def realize_plugin_query_extensions(source = Foreman::Plugin.graphql_types_registry.plugin_query_fields)
           source.map do |plugin_type|
-            send plugin_type[:field_type], plugin_type[:field_name], Foreman::Module.resolve(plugin_type[:type])
+            if plugin_type[:options].empty?
+              send plugin_type[:field_type], plugin_type[:field_name], Foreman::Module.resolve(plugin_type[:type])
```

Comment:
Why not apply_type as well? at least it would be the same.
```suggestion
              send plugin_type[:field_type], plugin_type[:field_name], apply_type(plugin_type[:type])
```